applications/luci-app-adblock/luasrc/view/adblock/runtime.htm

```diff
@@ -1,10 +1,192 @@
+				running(btn2_running, 0);
+			}
+			else if (input === "paused")
+			{
+				btn1.value = "Resume";
+				btn2.value = "Refresh";
+				btn1.disabled = false;
+				running(btn1_running, 0);
+				btn2.disabled = false;
+				running(btn2_running, 0);
+			}
+			else
+			{
+				btn1.value = "Suspend";
+				btn2.value = "Refresh";
+				btn1.disabled = true;
+				btn2.disabled = true;
+			}
+			view = document.getElementById("value_2");
+			input = json.data.adblock_version;
+			view.innerHTML = input || "-";
+			view = document.getElementById("value_3");
+			input = json.data.fetch_utility;
+			view.innerHTML = input || "-";
+			view = document.getElementById("value_4");
+			input = json.data.dns_backend;
+			view.innerHTML = input || "-";
+			view = document.getElementById("value_5");
+			input = json.data.overall_domains;
+			view.innerHTML = input || "-";
+			view = document.getElementById("value_6");
+			input = json.data.last_rundate;
+			view.innerHTML = input || "-";
+	}
+
+	function btn_action(action)
+	{
+		var btn1 = document.getElementById("btn1");
+		var btn1_running = document.getElementById("btn1_running");
+		var btn2 = document.getElementById("btn2");
+		var btn2_running = document.getElementById("btn2_running");
+
+		btn1.disabled = true;
+		btn2.disabled = true;
+
+		if (action.value === "Refresh")
+		{
+			running(btn2_running, 1);
+		}
+		else
+		{
+			running(btn1_running, 1);
+		}
+
+		new XHR.get('<%=luci.dispatcher.build_url("admin", "services", "adblock")%>/action/' + action.value, null,
+		function(x)
+		{
+			if (!x)
+			{
+				return;
+			}
+		});
+	}
+
+	function running(element, state)
+	{
+		if (state === 1)
+		{
+			var running_html = '<img src="<%=resource%>/icons/loading.gif" alt="<%:Loading%>" width="16" height="16" style="vertical-align:middle" />';
+			element.innerHTML = running_html;
+		}
+		else
+		{
+			element.innerHTML = '';
+		}
+	}
 
-<%+cbi/valueheader%>
+	XHR.get('<%=luci.dispatcher.build_url("admin", "services", "adblock", "status")%>', null,
+	function(x, json_info)
+	{
+		if (!x || !json_info)
+		{
+			var btn1 = document.getElementById("btn1");
+			var btn2 = document.getElementById("btn2");
+			btn1.value = "Suspend";
+			btn2.value = "Refresh";
+			btn1.disabled = true;
+			btn2.disabled = false;
+			return;
+		}
+		status_update(json_info)
+	});
 
-<input name="runtime" id="runtime" type="text" class="cbi-input-text" style="outline:none;border:none;box-shadow:none;background:transparent;color:#0069d6;font-weight:bold;line-height:30px;height:30px;width:20em;" value="<%=self:cfgvalue(section)%>" disabled="disabled" />
+	XHR.poll(5, '<%=luci.dispatcher.build_url("admin", "services", "adblock", "status")%>', null,
+	function(x, json_info)
+	{
+		if (!x || !json_info)
+		{
+			return;
+		}
+		status_update(json_info)
+	});
+//]]>
+</script>
 
-<%+cbi/valuefooter%>
+<h3>Runtime Information</h3>
+<div class="cbi-value" id="status_1">
+	<label class="cbi-value-title" for="status_1">Adblock Status</label>
```

Comment:
Same here, this seems to have been translated before, I just noticed that strings like `Adblock Status` got dropped with the translation refresh.